api/app/views/mno_enterprise/jpi/v1/admin/notifications/index.json.jbuilder

```diff
@@ -0,0 +1,34 @@
+json.notifications do
+  json.array! @tasks_to_be_reminded do |task|
+    json.object_id task.id
+    json.object_type 'task'
+    json.notification_type MnoEnterprise::Task::NOTIFICATION_TYPE::REMINDER
+    json.title 'Task Reminder'
+    json.message ["Title: #{task.title}",
```

Comment:
We should expose the task field and do the formatting in angular. Easier to customise/i18n